scripts/Diffraction/single_crystal/test/test_sxd.py

```diff
@@ -129,8 +129,10 @@ def test_integrate_peaks_MD_optimal_radius(self, mock_int_md):
 
         self.assertEqual(4, out.getNumberPeaks())
         self.assertEqual(3, mock_int_md.call_count)  # 2 peaks had similar radius and were integrated together
-        for icall, radius in enumerate([0.1021, 0.1319, 0.2519]):
-            self.assertAlmostEqual(radius, mock_int_md.call_args_list[icall].kwargs["PeakRadius"], delta=1e-3)
+        radius = [mock_int_md.call_args_list[icall].kwargs["PeakRadius"] for icall in range(mock_int_md.call_count)]
+        radius.sort()
+        for icall, expected_radius in enumerate([0.0948, 0.1148, 0.2519]):
```

Comment:
```suggestion
        for icall, expected_radius in enumerate([0.0948, 0.1148, 0.2348]):
```
0.23482148571955752 != 0.2519